### PR TITLE
feat: prep release

### PR DESCRIPTION
**Pull request by marrip**

### CODE REVIEW

File: workflow/envs/bcbio_variation_recall.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bcbio-variation-recall=0.2.6
+  - bcftools=1.11
```
Comment (Smeds) [inline workflow/envs/bcbio_variation_recall.yaml:7]: seems to differ from docker version

File: workflow/envs/bcftools.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bcftools=1.13
```
Comment (Smeds) [inline workflow/envs/bcftools.yaml:6]: common has bcftools 1.15

File: workflow/envs/bgzip.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 channels:
-  - conda-forge
   - bioconda
+  - conda-forge
   - defaults
 dependencies:
+  - htslib=1.10
```
Comment (Smeds) [inline workflow/envs/bgzip.yaml:6]: I guess bgzip will use common, is htslib=1.10 and tabix=1.11 same?

File: workflow/envs/fix_af.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - pysam=0.17.0
```
Comment (Smeds) [inline workflow/envs/fix_af.yaml:6]: if fix_af is using common container pysam version should be changed to 0.18.0

File: workflow/envs/freebayes.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - bcftools=1.13
```
Comment (Smeds) [inline workflow/envs/freebayes.yaml:6]: common container has version bcftools=1.14

File: workflow/envs/tabix.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - htslib=1.10
```
Comment (Smeds) [inline workflow/envs/tabix.yaml:6]: tabix in common container is tabix=1.11 . Will that match htslib=1.10?

File: workflow/envs/vt.yaml [hunk 7]
```diff
@@ -0,0 +1,7 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - cryptography=36.0.0
```
Comment (Smeds) [inline workflow/envs/vt.yaml:6]: should we sync version in our docker version which has cryptography=3.4.8

File: workflow/envs/bcbio_variation_recall.yaml (hunk 1 above)
Comment (marrip) [reply workflow/envs/bcbio_variation_recall.yaml:7]: just checked the version `bcftools 1.11`

File: workflow/envs/bgzip.yaml (hunk 3 above)
Comment (marrip) [reply workflow/envs/bgzip.yaml:6]: it seems, htslib/tabix/bgzip in `common:0.1.2` is `1.15.1` and not `1.11`

File: workflow/envs/fix_af.yaml (hunk 4 above)
Comment (marrip) [reply workflow/envs/fix_af.yaml:6]: done

File: workflow/envs/bcftools.yaml (hunk 2 above)
Comment (marrip) [reply workflow/envs/bcftools.yaml:6]: done

File: workflow/envs/tabix.yaml (hunk 6 above)
Comment (marrip) [reply workflow/envs/tabix.yaml:6]: changed to `1.15.1`

File: workflow/envs/vt.yaml (hunk 7 above)
Comment (marrip) [reply workflow/envs/vt.yaml:6]: done

**Review by jonca79 — APPROVED** [review]

File: .tests/compatibility/reference/WASH7P.fna [hunk 8]
```diff
@@ -0,0 +1,216 @@
+>chr1 WASH7P [organism=Homo sapiens] [GeneID=653635] [chromosome=1]
```
Comment (marrip) [inline .tests/compatibility/reference/WASH7P.fna:1]: why do we need the actual file when nothing is run? To keep the repo small, it should be an empty file.

File: .tests/compatibility/reference/WASH7P.dict [hunk 9]
```diff
@@ -0,0 +1,2 @@
+@HD	VN:1.6
```
Comment (marrip) [inline .tests/compatibility/reference/WASH7P.dict:1]: Same here, no actual file needed.

File: .tests/compatibility/reference/WASH7P.fna.amb [hunk 10]
```diff
@@ -0,0 +1 @@
+15009 1 0
```
Comment (marrip) [inline .tests/compatibility/reference/WASH7P.fna.amb:1]: same

File: .tests/compatibility/reference/WASH7P.fna.ann [hunk 11]
```diff
@@ -0,0 +1,3 @@
+15009 1 11
```
Comment (marrip) [inline .tests/compatibility/reference/WASH7P.fna.ann:1]: same

File: .tests/compatibility/reference/WASH7P.fna.fai [hunk 12]
```diff
@@ -0,0 +1 @@
+chr1	15009	68	70	71
```
Comment (marrip) [inline .tests/compatibility/reference/WASH7P.fna.fai:1]: same

**Review by marrip — COMMENTED** [review]
Please remove the reference files, it is unnecessary to have actual files there instead of empty ones for just a dry-run.

File: .tests/compatibility/reference/WASH7P.fna (hunk 8 above)
Comment (Smeds) [reply .tests/compatibility/reference/WASH7P.fna]: cleaned

File: .tests/compatibility/reference/WASH7P.fna.amb (hunk 10 above)
Comment (Smeds) [reply .tests/compatibility/reference/WASH7P.fna.amb]: cleaned

File: .tests/compatibility/reference/WASH7P.fna.ann (hunk 11 above)
Comment (Smeds) [reply .tests/compatibility/reference/WASH7P.fna.ann]: cleaned

File: .tests/compatibility/reference/WASH7P.fna.fai (hunk 12 above)
Comment (Smeds) [reply .tests/compatibility/reference/WASH7P.fna.fai:1]: this one we actually need=) It's used to decide fetch which chromosome will be used to merge files

Comment (marrip) [reply .tests/compatibility/reference/WASH7P.fna.fai:1]: true

**Review by Smeds — APPROVED** [review]

**Comment by Smeds** [comment]
Is it time to merge? @marrip  @jonca79

**Comment by jonca79** [comment]
Ok for me!

**Pull request merged**